### PR TITLE
File input docs were upgraded

### PR DESCRIPTION
**Pull request by DmitriiBabakaevTR**
File Input docs were upgraded

### CODE REVIEW

File: source/includes/documentation/_vue_js_elements.md [hunk 1]
```diff
@@ -1413,67 +1413,107 @@ For examples of usage see: [JDI Vuetify Select tests](https://github.com/jdi-tes
...
+**getAppendOuterIcon()** | Returns first append outer icon            | Icon
+**getText()** | Returns text                               | String
+**label()** | Returns label                              | Label
+**placeholder()** | Returns placeholder text                   | String
+**setText(String text)** | Set text                                   | void
+**sendKeys(CharSequence... value)** | Add text                                   | void
+**clear()** | Clear text area                            | void
+**focus()** | Set mouse to text area                     | void
+**isDisplayed()** | Check if element is displayed              | boolean
+**isDisabled()** | Check if element is disabled               | boolean
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:1501]: why 'isDisabled' repeated 2 times? :)

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md (hunk 1 above)
Comment (DmitriiBabakaevTR) [reply source/includes/documentation/_vue_js_elements.md:1501]: not 2 time
1- isDisplayed
2- isDisabled

Comment (MayaElf) [reply source/includes/documentation/_vue_js_elements.md:1501]: oh) thnx

File: source/includes/documentation/_vue_js_elements.md [hunk 2]
```diff
@@ -1413,67 +1413,107 @@ For examples of usage see: [JDI Vuetify Select tests](https://github.com/jdi-tes
...
+  chipsFileInput.show();
+  chipsFileInput.is().multiple();
+  chipsFileInput.uploadFiles(asList(pathTXT.toString(), pathPNG.toString()));
+  chipsFileInput.has().files(asList(pathTXT.getFileName().toString(), pathPNG.getFileName().toString()));
+  smallChipsFileInput.show();
+  smallChipsFileInput.is().multiple();
+  smallChipsFileInput.uploadFile(pathTXT.toString());
+  smallChipsFileInput.has().file(pathTXT.getFileName().toString());
+  smallChipsFileInput.uploadFile(pathPNG.toString());
+  smallChipsFileInput.has().files(asList(pathTXT.getFileName().toString(), pathPNG.getFileName().toString()));
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:1442]: закрывающая скобка?)

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md (hunk 2 above)
Comment (DmitriiBabakaevTR) [reply source/includes/documentation/_vue_js_elements.md]: fixed

**Review by MayaElf — APPROVED** [review]

**Review by vklonin — APPROVED** [review]

**Pull request merged**